gen3-integration-tests/tests/test_graph_submit_and_query.py

```diff
@@ -60,42 +95,436 @@ def test_submit_query_and_delete_records(self):
+        gen3_indexd = Gen3Index(auth_provider=indexd_auth)
+        indexd = Indexd()
+        records = gen3_indexd.get_all_records()
+
+        # Delete indexd records
+        for record in records:
+            gen3_indexd.delete_record(guid=record["did"])
+
+        file_record = self.sd_tools.get_file_record()
+        file_record.props["consent_codes"] += ["CC1", "CC2"]
+        self.sd_tools.submit_links_for_node(record=file_record)
+        self.sd_tools.submit_record(record=file_record)
+
+        file_record.indexd_guid = self.sd_tools.get_indexd_id_from_graph_id(
+            unique_id=file_record.unique_id
+        )
+
+        file_node_wit_ccs = FileNode(
+            did=file_record.indexd_guid,
+            props={
+                "md5sum": file_record.props["md5sum"],
+                "file_size": file_record.props["file_size"],
+            },
+            authz=["/consents/CC1", "/consents/CC2"],
+        )
+
+        # Verify indexd record was created with the correct consent codes
+        response = indexd.get_record(file_node_wit_ccs.did)
+        indexd.file_equals(response, file_node_wit_ccs)
+
+    @pytest.mark.indexd
+    @pytest.mark.portal
+    @pytest.mark.graph_query
+    def test_file_landing_page(self, page: LoginPage):
+        """
+        Scenario: Test file landing page
+        Steps:
+            1. Submit graph data using sheepdog. Verify records are present.
+            2. Create a file record
+            3. Get core metadata using record in step 2
+            4. Reteive object_id from metadata record recieved
+            5. Load metadata page using object_id and verify the elements
+            6. Delete file record and delete indexd record
+            7. Delete all records. Verify all records are deleted.
+        """
+        login_page = LoginPage()
+        files_landing_page = FilesLandingPage()
+        file_record = self.sd_tools.get_file_record()
+        self.sd_tools.submit_links_for_node(file_record)
+        self.sd_tools.submit_record(record=file_record)
+        file_record.indexd_guid = self.sd_tools.get_indexd_id_from_graph_id(
+            unique_id=file_record.unique_id
+        )
+        metadata = self.sd_tools.get_core_metadata(
+            file=file_record, user="main_account"
+        )
+        self.sd_tools.verify_metadata_json_contents(
+            record=file_record, metadata=metadata
+        )
+
+        # Perform login and logout operations using main_account to create a login record for audit service to access
+        logger.info("Logging in with mainAcct")
+        login_page.go_to(page)
+        login_page.login(page)
+        files_landing_page.goto_metadata_page(page, metadata.json()["object_id"])
+        files_landing_page.verify_metadata_page_elements(page)
+        login_page.logout(page)
+
+        self.sd_tools.delete_record(unique_id=file_record.unique_id)
+
+    @pytest.mark.indexd
+    @pytest.mark.graph_query
+    def test_sheepdog_metadata(self):
```

Comment:
`test_sheepdog_metadata` not a good name now that the core metadata tests are in the sheepdog test suite. rename all the `test_sheepdog_metadata` tests